### ISSUE TITLE
Incorrect blocking large files on proposal attachments

### ISSUE THREAD

**Issue by fernandoabolafio** [decred/politeiagui#312]
- go to proposals/new
- Click on Attach file
- Select a file which exceeds the max allowed size and click ok.
It will correctly be blocked and an alert will show up.

now:
- Click to attach a file
- Select multiple files, including the one which exceeds the maximum size
It will correct alert about the large size file. However, it always wrongly blocks the first selected file (by alphabetical order). 

In summary: selecting one file per time works just fine, selecting multiple files at once breaks it.

**Comment by fernandoabolafio** [comment]
I'm on it.
